core/src/main/java/com/radixdlt/consensus/sync/BFTSync.java

```diff
@@ -246,10 +250,11 @@ public SyncResult syncToQC(HighQC highQC, @Nullable BFTNode author) {
       return SyncResult.SYNCED;
     }
 
-    // TODO: Move this check into pre-check
+    // TODO: Check for other conflicting QCs
```

Comment:
Conflicting QCs would occur before a fork in the ledger. We would need to keep track of QC produced per round, we can then catch any conflicting QCs. Will add more comments!

Reply:
Oh nice, okay. This could relate to the issue we're seeing with BFT Sync? Although I think that issue is possibly more likely fixed by Lukasz' changes